docs/developer-guide/table-functions.rst

```diff
@@ -0,0 +1,90 @@
+.. _table-functions:
+
+
+Using Table Functions With KSQL
+###############################
+
+A _table function_ is a function that returns a set of zero or more rows. Contrast this to a scalar
+function which returns a single value.
+
+Table functions are analagous to the `FlatMap` operation commonly found in
+functional programming or stream processing frameworks such as [kstreams].

```

Comment:
```suggestion
functional programming or stream processing frameworks such as |kstreams|.
```

Reply:
|kstreams| is a [string token](https://github.com/confluentinc/docs/blob/655c9086d6409b2547c8402cf90482995bc5b69a/conf.py#L355) that gets replaced during the doc build.

Reply:
done